Mathmagician.Tests/PrimeTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace Mathmagician.Tests
+{
+    [TestClass]
+    public class PrimeTest
+    {
+        [TestMethod]
+        public void Prime0()

```

Comment:
@VikingPaul This test needs to be named better. What is it testing? I should know by simply reading the test method name.